### PR TITLE
etcdctl\ctlv3: updated readme.md for timetolive example

### PR DESCRIPTION
**Pull request by sinsharat**
added output section which was missing for lease timetolive example

### CODE REVIEW

File: etcdctl/README.md [hunk 1]
```diff
@@ -344,13 +344,16 @@ LEASE TIMETOLIVE retrieves the lease information with the given lease ID.
...
+#OK
+
 ./etcdctl put foo2 bar --lease=2d8257079fa1bc0c
+#OK
 
 ./etcdctl lease timetolive 2d8257079fa1bc0c
 # lease 2d8257079fa1bc0c granted with TTL(500s), remaining(481s)
 
 ./etcdctl lease timetolive 2d8257079fa1bc0c --keys
-# lease 2d8257079fa1bc0c granted with TTL(500s), remaining(472s), attached keys([foo2 foo1])
```
Comment (gyuho) [inline etcdctl/README.md]: `--keys` returns unordered list of keys, so let's just leave it as it is.

File: etcdctl/README.md [hunk 2]
```diff
@@ -344,13 +344,16 @@ LEASE TIMETOLIVE retrieves the lease information with the given lease ID.
 # lease 2d8257079fa1bc0c granted with TTL(500s)
 
 ./etcdctl put foo1 bar --lease=2d8257079fa1bc0c
+#OK
+
 ./etcdctl put foo2 bar --lease=2d8257079fa1bc0c
+#OK
```
Comment (gyuho) [inline etcdctl/README.md]: Insert single whitespace character `# OK`?

File: etcdctl/README.md [hunk 3]
```diff
@@ -344,13 +344,16 @@ LEASE TIMETOLIVE retrieves the lease information with the given lease ID.
 # lease 2d8257079fa1bc0c granted with TTL(500s)
 
 ./etcdctl put foo1 bar --lease=2d8257079fa1bc0c
+#OK
```
Comment (gyuho) [inline etcdctl/README.md]: Insert single whitespace character `# OK`?

File: etcdctl/README.md (hunk 2 above)
Comment (sinsharat) [inline etcdctl/README.md]: @gyuho sure will modify the changes.

File: etcdctl/README.md (hunk 3 above)
Comment (sinsharat) [inline etcdctl/README.md]: @gyuho sure will modify the changes.

File: etcdctl/README.md (hunk 1 above)
Comment (sinsharat) [inline etcdctl/README.md]: @gyuho I had one query. Actually i have just modified the way they are displayed and i haven't removed it, just re-ordered since i was getting the result in order of the keys they were created. So just wanted to confirm if you still wanted to change it back to the was it was? since its returing the order in sorted manner.
Request you to feedback on this. Will modify if you feel that way.
Thank you.

Comment (sinsharat) [inline etcdctl/README.md]: @gyuho ok i got your point since it unordered you want to keep it that way, to make it more specific. Will revert the changes.
Thanks for your input.

Comment (gyuho) [inline etcdctl/README.md]: @sinsharat Yes please. We don't sort keys in response unless specified.

Thanks!

If you want to look at the source code

https://github.com/coreos/etcd/blob/master/lease/lessor.go#L483-L490

**Comment by gyuho** [comment]
LGTM. Thanks!

**Comment by sinsharat** [comment]
@gyuho thank you for your valuable commments.
